int32 ts
float32 q0
float32 q1
float32 q2
float32 q3
float32 wx
float32 wy
float32 wz